.changeset/friendly-nails-push.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': minor
+---
+
+`ERC1363`: add `ERC1363` implementation other than `IERC1363Errors`, `ERC1363Holder` and tests.
```

Comment:
The tests are not relevant for the changelog imo.

```suggestion
`ERC1363`: Add implementation of the token payable standard allowing execution of contract code after transfers and approvals, along with its `ERC1363Receiver` and `ERC1363Spender` extensions. 
```

Reply:
I'm assuming we'll provide 2 extensions instead of one

Reply:
```suggestion
`ERC1363`: Add implementation of the token payable standard allowing execution of contract code after transfers and approvals.
```

IMO `ERC1363Receiver` and `ERC1363Spender` should just be the interfaces (that are alredy in the codebase) and the mock (that are not relevant here)

Reply:
> I'm assuming we'll provide 2 extensions instead of one

Why would we provide 2 extensions ?

Reply:
> Why would we provide 2 extensions ?

I thought it was better than 1 including both behaviors. Now I agree we only should include interfaces

Reply:
You meant 2 "*Holder" contract?

I thing the ambiguity is that I would personnaly not call these extensions. To me `ERC721Pausable` is an extension of `ERC721` ... but `ERC721Holder` is not an extension.

Reply:
Yes, that's why I meant.